### ISSUE TITLE
[TF2] Players can enter the reference pose / A-Pose / T-Pose by using and picking up certain weapons.

### ISSUE THREAD

**Issue by gurfan** [ValveSoftware/Source-1-Games#4123]
By using and picking up certain weapons, players can enter the reference pose indefinitely. 

There are multiple ways to achieve this. Here are a few:

- Throw mad milk/jarate. While still holding primary fire, pick up a new mad milk/jarate from the ground.
- Activate a buff banner/concherer/battalion's backup, hold primary fire, equip the B.A.S.E. jumper, and touch a resupply cabinet.
- Pick up a minigun from the ground, empty its ammo, and pick up a new minigun while still holding primary fire.

More methods to recreate this bug are explained on the TF2 Wiki:
https://wiki.teamfortress.com/wiki/Civilian#The_reference_pose

**Comment by MicRouSN7** [comment]
on god bro you're out here to ruin people's enjoyment of the game in any way possible huh

**Comment by AzureAzel** [comment]
> on god bro you're out here to ruin people's enjoyment of the game in any way possible huh

With all due respect, this is a forum for reporting bugs. It doesn't matter what the context is, or if it's funny or not. A bug is a bug. Although this is relatively harmless, I don't see how they're doing anything wrong by reporting a bug from the game on a forum for reporting bugs for the game.

**Comment by MicRouSN7** [comment]
> > on god bro you're out here to ruin people's enjoyment of the game in any way possible huh
> 
> With all due respect, this is a forum for reporting bugs. It doesn't matter what the context is, or if it's funny or not. A bug is a bug. Although this is relatively harmless, I don't see how they're doing anything wrong by reporting a bug from the game on a forum for reporting bugs for the game.

I agree. Just frustrated by their decision to report like two of the most harmless bugs instead of actual harmful ones like the contracker one. I'm going to be posting about that one today when I get home because that one can actually obscure your head hitbox and let you move around and stuff

**Comment by thejaviertc** [comment]
> > > on god bro you're out here to ruin people's enjoyment of the game in any way possible huh
> > 
> > 
> > With all due respect, this is a forum for reporting bugs. It doesn't matter what the context is, or if it's funny or not. A bug is a bug. Although this is relatively harmless, I don't see how they're doing anything wrong by reporting a bug from the game on a forum for reporting bugs for the game.
> 
> I agree. Just frustrated by their decision to report like two of the most harmless bugs instead of actual harmful ones like the contracker one. I'm going to be posting about that one today when I get home because that one can actually obscure your head hitbox and let you move around and stuff

Is not reporting it because is already reported in #4065

**Comment by Ashetf2** [comment]
If Valve ever mind in patch this bug, maybe they could make an obtainable by achievement "A pose" taunt
